### PR TITLE
adding ext-router entity definition and metrics

### PR DESCRIPTION
**Pull request by thezackm**
### Relevant information

adding `ROUTER` entity for the `EXT` domain. Original definition is based on network device metrics sent via Kentik technical partnership, but this definition is being kept ambiguous to allow for other technical partnerships to use the entity in the future.

`ROUTER` relates to any L3 device in the networking space that has the capability to route packets based on defined protocols. This entity can also be decorated with Flow telemetry (NetFlow/sFlow/jFlow/IPFIX/etc), among other things. 

### Checklist

* [X] I've read the guidelines and understand the acceptance criteria.
* [X] The value of the attribute marked as `identifier` will be unique and valid. 
* [X] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

File: definitions/ext-router/definition.yml
Comment (IreneMLC) [inline definitions/ext-router/definition.yml:10]: we'll only really apply 1 of these conditions atm (the telemetry won't have to match both), right @srvaroa?

Comment (thezackm) [reply definitions/ext-router/definition.yml:10]: this telemetry's current design will require matching both; if we can't do that please let me know ASAP so I can go back to the vendor and request a change. thanks!!!

File: definitions/ext-router/golden_metrics.yml
Comment (srvaroa) [inline definitions/ext-router/golden_metrics.yml:6]: Same as in the other PR, I think this where is not necessary (pending for @naxhh  to confirm)

**Review by srvaroa — COMMENTED** [review]
LGTM! thanks for adding these (I'll wait for @naxhh's reply on a minor comment before merging)

File: definitions/ext-router/definition.yml
Comment (IreneMLC) [reply definitions/ext-router/definition.yml:10]: Thaks for the change!

**Review by IreneMLC — APPROVED** [review]

File: definitions/ext-router/summary_metrics.yml
Comment (naxhh) [inline definitions/ext-router/summary_metrics.yml:11]: The `as` here will not be really used anywhere. And I'm not sure how our summary metrics will handle that.
@gvassallo would appreciate your input on this one.

**Review by naxhh — COMMENTED** [review]
Aside of the comment LGTM

File: definitions/ext-router/summary_metrics.yml
Comment (gvassallo) [reply definitions/ext-router/summary_metrics.yml:11]: i think this might break summary metrics or in general, it's not required

Comment (thezackm) [reply definitions/ext-router/summary_metrics.yml:11]: @gvassallo | @naxhh - so I should remove all of the `'AS'` on these PRs for summary metrics only, or both summary and golden?

Comment (gvassallo) [reply definitions/ext-router/summary_metrics.yml:11]: I'd say so, we're gonna add the `as` to the golden metric queries by default in one of our next releases. I'd leave @naxhh making the final decision though :D

Comment (naxhh) [reply definitions/ext-router/summary_metrics.yml:11]: @thezackm remove them all from both sides. We are in the process of automating this and will be easier for us.
Thanks for the patience!


Side note: I'll make a note to check SM behavior on this so we can add it to the linting.

Comment (thezackm) [reply definitions/ext-router/summary_metrics.yml]: done and done. thanks!

**Review by naxhh — APPROVED** [review]

**Pull request merged**
